gradio/components/multimodal_textbox.py

```diff
@@ -74,7 +74,7 @@ def __init__(
         render: bool = True,
         text_align: Literal["left", "right"] | None = None,
         rtl: bool = False,
-        submit_btn: str | Literal[False] = "⌲",
+        submit_btn: str | Literal[False] = "",
```

Comment:
Better to use `None` I think? And we should mention in the docstring that if None, an svg of a send icon is used.
```suggestion
        submit_btn: str | Literal[False] | None = None,
```